demos/tests/run_tests.py

```diff
@@ -338,8 +338,7 @@ def option_to_args(key, value):
                                 exit_msg = f'Exit code: {e.returncode}\n'
                             elif isinstance(e, subprocess.TimeoutExpired):
                                 exit_msg = f'Command timed out after {e.timeout} seconds\n'
-                            output += exit_msg
-                            print(output)
+                            print('{}\n{}'.format(output, exit_msg))
```

Comment:
`output` var is used to save log file below: `write_log(output, args.log_file)`. So you need this `output += exit_msg`.

If I understand correctly, `TimeoutExpired.output` is of bytes type for you. Maybe you can cast it to str before that `+=`

Please, add a comment to code with link to a bug https://bugs.python.org/issue43431 to explain why we have such cast.

Reply:
> If I understand correctly, `TimeoutExpired.output` is of bytes type for you. Maybe you can cast it to str before that `+=`
> 
Yes, you are right. `output `var is bytes type under that host env when timeout happened. While don't know why the `output ` is Str type on my host when timeout happened.

Reply:
Check the bug. It appears on python3.8+ may be you have python3.7 or 3.6 on your host